internal/backends/postgresql/metadata/metadata.go

```diff
@@ -15,21 +15,184 @@
 // Package metadata provides access to databases and collections information.
 package metadata
 
-import "github.com/FerretDB/FerretDB/internal/types"
+import (
+	"errors"
+	"fmt"
+	"slices"
+
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/iterator"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+)
+
+const (
+	// DefaultColumn is a column name for all fields.
+	DefaultColumn = "_jsonb"
+
+	// IDColumn is a PostgreSQL path expression for _id field.
+	IDColumn = DefaultColumn + "->'_id'"
+)
 
 // Collection represents collection metadata.
 type Collection struct {
-	Name      string
-	TableName string
-	// TODO indexes, etc.
+	Name      string `json:"_id"`
+	TableName string `json:"table"`
+	Settings  Settings
+}
+
+// deepCopy returns a deep copy.
+func (c *Collection) deepCopy() *Collection {
+	if c == nil {
+		return nil
+	}
+
+	return &Collection{
+		Name:      c.Name,
+		TableName: c.TableName,
+		Settings:  c.Settings.deepCopy(),
+	}
+}
+
+// Settings represents collection settings.
+type Settings struct {
+	Indexes []IndexInfo `json:"indexes"`
```

Comment:
Let's remove _everything_ about indexes for now

Reply:
Sure!